src/components/stage-wrapper/stage-wrapper.css

```diff
@@ -1,7 +1,7 @@
 @import "../../css/units.css";
 @import "../../css/colors.css";
 
-.stage-wrapper {
+.stage-wrapper * {
     box-sizing: border-box;
```

Comment:
Should this be `.stage-wrapper, .stage-wrapper *` so the `.stage-wrapper` itself retains the previous `box-sizing`?  Also, is this in this PR intentionally?

Reply:
It's actually irrelevant whether stage-wrapper has box-sizing set. The important part was all the children - which was missing.